policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_payment(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        eligibility = household("qc_fa_eligibility", period)
+
+        # check if the household has cohabiting spouse
+        has_spouse = household("is_married", period)
+
+        # eligible dependent child - under 18 years of age
+        person = household.members
+        age_eligible = person("age", period) < p.age_eligibility
+
+        # check if each child is full custody
+        full_custody = person("full_custody", period)
+        shared_custody_multiplier = where(
+            full_custody, 1, p.shared_custody_multiplier
+        )
+
+        # method 1: calculate using max amount with reduction threshold, (C + D) − 4% (E − F) in taxation axt
```

Comment:
please keep comments to 79 characters if possible
```suggestion
        # method 1: calculate using max amount with reduction threshold,
        # (C + D) − 4% (E − F) in taxation act
```